tcs/csp_solver_core.h

```diff
@@ -852,6 +840,21 @@ class C_csp_solver
 		}
 	};
 
+    struct S_op_mode_params
+    {
+        int m_cr_mode;      //[-] Collector-receiver operating mode
+        int m_pc_mode;      //[-] Power cycle operating mode
+        int m_solver_mode;  //[-] Component connection config - see C_MEQ__m_dot_tes
+        int m_step_target_mod;  //[-] What is variable timestep targeting?
+        bool m_is_defocus;  //[-] Does timestep include defocus?
+
+        S_op_mode_params()
+        {
+            m_cr_mode = m_pc_mode = m_solver_mode = m_step_target_mod = -1;
+        m_is_defocus = false;
+        }
+    };
```

Comment:
Since this all looks new, remove m_ prefixes. Also, the modes should be enumerations, specifically enumeration classes, instead of ints.  This should also eventually be the case for all other modes.
Clarify step_target_mod.

Reply:
it is not time